vignettes/min_workflow_local.Rmd

```diff
@@ -0,0 +1,165 @@
+---
+title: "A minimalist example with local data"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{A minimalist example with local data}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+## Goal:
+
+Build, deploy and access a(n overly) simple data product locally to get 
+familiarized with the concepts and functions. This method is applicable for
+building data products on locally or on mounted drives. 
+
+## User story
+
+We are interested in a data product that simply provides distances in cars
+dataset in metric unit. See `?cars` for additional detail about the dataset.
+
+## Step 1: Initialize the project
+
+As this is a new project, we initialize the project using `dpbuild::dp_init`. See
+the getting started [vignette](https://amashadihossein.github.io/daapr/articles/daapr.html) for details of what the initialization does.
+
+```{r dp_init, eval=F}
+library(daapr)
+
+# Dry function call to set board location and name
+board_params_set_dried <- fn_dry(board_params_set_local(board_alias = "<ALIAS>",
+                                                        folder = "<FOLDER_PATH>"))
+
+# Dry function call to setting credentials -- none needed for local board
+# TODO: add option for no credentials, this is a placeholder
+creds_set_dried <- fn_dry(2)
```

Comment:
I'm not sure this will work with the credential checking that happens in the update to `fn_dry`. Could dp_init be updated to take NULL credentials for local boards?